packages/amplication-data-service-generator/src/server/resource/service/create-service.ts

```diff
@@ -375,6 +389,23 @@ async function createToManyRelationFile(
   return toManyFile;
 }
 
+function identityParentIdType(entity: Entity): namedTypes.Identifier {
+  const relatedEntityFiledId = entity.fields?.find(
+    (relatedEntityField) => relatedEntityField.dataType === EnumDataType.Id
+  );
+
+  const { idType } = (relatedEntityFiledId?.properties as types.Id) || "CUID";
+  const idTypeTSOptions: {
+    [key in types.Id["idType"]]: namedTypes.Identifier;
+  } = {
+    AUTO_INCREMENT: builders.identifier("number"),
+    UUID: builders.identifier("string"),
+    CUID: builders.identifier("string"),
+  };
+
+  return idTypeTSOptions[idType] ?? idTypeTSOptions["CUID"];
```

Comment:
what is this for (idTypeTSOptions["CUID"]) ?

Reply:
for the default

Reply:
I logged the idType and for the entities that don't have this property, it returns `undefined`
![image](https://user-images.githubusercontent.com/39680385/208285093-b6c76bfb-f1dd-4a6e-b60f-a9426cd167a0.png)